hw/top_darjeeling/lint/top_darjeeling.waiver

```diff
@@ -8,19 +8,43 @@
 set_reset_drivers prim_clock_mux2 prim_flop_2sync prim_flop
 set_clock_drivers prim_clock_buf prim_clock_mux2
 
+# TODO: remove these waivers later when the USB clock was removed from the integrated system
+# in a clean way. Right now there are some dangling signals that we have to keep in order
+# to not disrupt the RTL structure and associated DV.
+waive -rules {HIER_BRANCH_NOT_READ INPUT_NOT_READ} -location {xbar_main.sv} \
+      -regexp {'(clk_usb_i|rst_usb_ni)' is not read from in module 'xbar_main'} \
+      -comment "Temporary waiver for dangling USB clocks and resets."
+
+# All leaf resets have a reset multiplex
+waive -rules RESET_MUX -location {top_darjeeling.sv} -regexp {Asynchronous reset .*rstmgr_aon_resets\.rst.* is driven by a multiplexer} \
+      -comment "This is dedicated reset infrastructure, and hence permissible"
+
 # scan reset is a legal asynchronous reset
-waive -rules RESET_USE -location {top_earlgrey.sv} -regexp {'scan_rst_ni' is connected to .* port 'scan_rst_ni', and used as an asynchronous reset or set 'rst_ni' at} \
+waive -rules RESET_USE -location {top_darjeeling.sv} -regexp {'scan_rst_ni' is connected to .* port 'scan_rst_ni', and used as an asynchronous reset or set 'rst_.*ni' at} \
       -comment "Scan reset is a legal asynchronous reset"
 
+waive -rules {HIER_BRANCH_NOT_READ} -location {pinmux.sv} \
+      -msg {Net 'rst_sys_ni' is not read from in module 'pinmux'} \
+      -comment "This signal is unused but cannot yet be gated in ipgen"
+
+waive -rules {HIER_BRANCH_NOT_READ} -location {pinmux.sv} \
+      -msg {Net 'scanmode_i' is not read from in module 'pinmux'} \
+      -comment "This signal is unused but cannot yet be gated in ipgen"
+
+waive -rules INPUT_NOT_READ -location {pinmux.sv} -regexp {Input port '(rst_sys_ni|scanmode_i)' is not read from} \
+      -comment "This signal is unused but cannot yet be gated in ipgen."
```

Comment:
I think we should track the mentioned pinmux ipgen gap in an issue and link it here, to make sure we remove these waivers once pinmux ipgen supports it